docs/source/model_design/general_research.rst

```diff
@@ -679,11 +679,131 @@ Conducting Systematic Literature Searches/Reviews
+
+.. image:: acyclic_dags.svg
+
+Applications, Assumptions, and Limitations of Directed Acyclic Graphs
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+DAGs are used in epidemiology to represent a network of relationships between several variables, which in turn helps to guide the selection of variables that will need to be measured or considered in a given study/project. Further, the exact relationship between variables in a DAG will dictate *how* to treat or consider a specific variable in that study/project by helping to indentify potential confounding, intermediate, or mediating variables between a given exposure and outcome of interest (see the `Causal Relationships`_ section for definitions).
+
+However, DAGs are limited in that they require *prior causal knowledge* about the variables that may be causally related to a given exposure and outcome of interest. Therefore, there may be uncertain, unmeasured, or unknown variables that cannot be represented in a given DAG. Further, DAGs do not allow for easy representation of direction or magnitude of a causal relationship or for effect modification.
+
+For the purpose of this section of our documentation, we will assume that each DAG is completely accurate, represents **all** relevant variables in a given causal network and that there are no unrepresented variables.
+
+Types of Pathways Within Directed Acyclic Graphs
+++++++++++++++++++++++++++++++++++++++++++++++++
+
+A path in a DAG is defined as any route drawn through arrows that link two variables **regardless of the direction of the arrows**.
+
+Directed Paths
+~~~~~~~~~~~~~~
+
+Directed paths are those in which variables are connected through arrows that flow all flow in the same direction. See below for an example of a directed path between variables A and D.
+
+.. image:: directional_dag.svg
+
+**In the case of a directed path between two variables, those variables will be causally related.**
+
+Undirected Paths
+~~~~~~~~~~~~~~~~
+
+Undirected paths are those in which variables are connected through arrows that do not all flow from head to tail. The two cases of undirected paths covered here include backdoor paths and colliders.
+
+**In the case that there is an undirected path (and no directed path) bewteen two variables, those variables will NOT be causally related, although they may be associated in some way.**
+
+Backdoor Paths
+^^^^^^^^^^^^^^
+
+Backdoor paths are those in which two variables are connected via a common cause. In other words, two variables that are linked via a variable that has two arrows flowing *away* from it. See below for an example in which variables Y and Z are connected via a backdoor path through variable X.
+
+.. image:: backdoor_path_dag.svg
+
```

Comment:
I like this - I had seen backdoor paths mentioned in papers but didn't know what they were. This made it really clear.

I think to make it even clearer, it would be good to add one more sentence after the image clarifying that a backdoor path can have more than one variable between the endpoints. E.g. if we added some more variables between X and Z, it would still count as a backdoor path between Y and Z.

Reply:
Actually, based on [this reference (pp. 6-7)](https://ftp.cs.ucla.edu/pub/stat_ser/r332.pdf), it seems that the key property of a backdoor path is that it has to start with an arrow pointing into Y. This is not necessarily implied by your definition (e.g. put a collider between X and Y in your diagram), although I think it *is* implied by your definition of *open backdoor path* below - if the path has no colliders *and* has arrows pointing in different directions, then both the initial and final arrows must be pointing "outward" instead of "inward."

Reply:
I think perhaps to resolve both these issues, you could start by saying something like:

> Backdoor paths are those in which two variables are connected via a common cause. The simplest example is when two variables are linked via a third variable that has two arrows flowing *away* from it. See below for an example in which variables Y and Z are connected via a backdoor path through variable X.

Then show the diagram, then afterwards, give a more general definition of backdoor paths.